source/wp-content/themes/wporg-developer-blog/parts/footer.html

```diff
@@ -0,0 +1,63 @@
+    <!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|60"}}}} -->
+    <div class="wp-block-columns alignwide">
+        <!-- wp:column {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}}} -->
+        <div class="wp-block-column">
+            <!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}},"fontSize":"large"} -->
+            <h2 class="wp-block-heading has-large-font-size" style="font-style:normal;font-weight:400">Have an idea for a new post?</h2>
+            <!-- /wp:heading -->
+
+            <!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|30","left":"var:preset|spacing|40"}}}} -->
+            <div class="wp-block-columns">
+                <!-- wp:column -->
+                <div class="wp-block-column">
+                    <!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"default"}} -->
+                    <div class="wp-block-group">
+                        <!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-2"}}},"typography":{"fontStyle":"normal","fontWeight":"700"}},"textColor":"blueberry-2","fontSize":"normal","fontFamily":"inter"} -->
+                        <h3 class="wp-block-heading has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700"><strong><a href="https://developer.wordpress.org/news/how-to-contribute/">Learn how to contribute</a></strong></h3>
+                        <!-- /wp:heading -->
+
+                        <!-- wp:paragraph {"fontSize":"normal"} -->
+                        <p class="has-normal-font-size">Share your knowledge with fellow WordPress developers.</p>
+                        <!-- /wp:paragraph -->
+                    </div>
+                    <!-- /wp:group -->
+                </div>
+                <!-- /wp:column -->
+
+                <!-- wp:column -->
+                <div class="wp-block-column">
+                    <!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"default"}} -->
+                    <div class="wp-block-group">
+                        <!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-2"}}}},"textColor":"blueberry-2","fontSize":"normal","fontFamily":"inter"} -->
+                        <h3 class="wp-block-heading has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size"><strong><a href="https://developer.wordpress.org/news/tips-and-guidelines-for-writers/">Review tips and guidelines</a></strong></h3>
+                        <!-- /wp:heading -->
+
+                        <!-- wp:paragraph -->
+                        <p>Everything you need to know about writing for the Blog.</p>
```

Comment:
As an FYI, to translate content, we need to move templates that include strings to pattern templates and use localization functions.

Reply:
Ah good call. I'll get this fixed up today.

Reply:
This is fixed in the latest commits.